training/templates/create_event.html

```diff
@@ -147,10 +173,28 @@
                     $('#id_host_college').html(data);
                 }
               });
+	}	
    	});
 
-
+	$('#id_event_type').change(function(){		
+		var event_type = $(this).val();
+		// Change visibility of college, company & city based on event type
+		$('#company').toggle(event_type === "PDP");
+		$('#city').toggle(event_type === "PDP" || event_type === "CDP");
+		$('#institute').toggle(event_type !== "PDP" && event_type !== "CDP");
+
+		// Set default college [IIT Bombay (academic id : 621)] and fees if event type is PDP or CDP
+		const host_college = $('#id_host_college');
+		const event_fee = $('#id_event_fee')
+		if(event_type === "PDP"){
+			event_fee.val(1000);
+			host_college.val(621); 
+
+		}else if(event_type === "CDP"){
+			event_fee.val(500);
```

Comment:
Is this amount on the python side also? If possible, get the fee from the backend.